lib/src/atomic/_stacks-misc.less

```diff
@@ -99,18 +99,30 @@
 //  ============================================================================
 //  $  OPACITY
 //  ----------------------------------------------------------------------------
-.o0                       { opacity: 0 !important; }
-.o5                       { opacity: 0.05 !important; }
-.o10                      { opacity: 0.1 !important; }
-.o20                      { opacity: 0.2 !important; }
-.o30                      { opacity: 0.3 !important; }
-.o40                      { opacity: 0.4 !important; }
-.o50                      { opacity: 0.5 !important; }
-.o60                      { opacity: 0.6 !important; }
-.o70                      { opacity: 0.7 !important; }
-.o80                      { opacity: 0.8 !important; }
-.o90                      { opacity: 0.9 !important; }
-.o100                     { opacity: 1 !important; }
+.o-0                       { opacity: 0 !important; }
```

Comment:
I don't think these need to be renamed, do they? What's with adding the dash?

Reply:
For some reason, these will not work properly with the `\:` operator when creating the hover states without it ~ any ideas why that is? I'm not familiar with using that.

Reply:
Maybe it will work and i just didn't have the correct syntax in place? Let me know when you can chat and we can discuss before I keep moving forward.